realm/realm-library/src/main/java/io/realm/RealmObject.java

```diff
@@ -242,6 +242,46 @@ public final boolean isLoaded() {
     }
 
     /**
+     * Checks if this object is managed by Realm. A managed object is just a wrapper around the data in the underlying
+     * Realm file. On Looper threads, a managed object will be live-updated so it always points to the latest data. It
+     * is possible to register a change listener using {@link #addChangeListener(RealmChangeListener)} to be notified
+     * when this happens. Managed objects are thread confined so the they cannot be accessed from other threads than
+     * the one that created them.
+     *
+     * If this method returns {@code false}, the object is unmanaged. An unmanaged object is just a normal Java object,
+     * so it can be parsed freely across threads, but the data in the object is not connected to the underlying data,
+     * so it will not be live updated.
+     *
+     * It is possible to convert a object from managed to unmanaged using {@link Realm#copyFromRealm(RealmModel)}.
```

Comment:
`to create a managed object from an unmanaged object.`
technically it is not converting, the original object stays the same.